hirefire/procs/celery.py

```diff
@@ -1,22 +1,31 @@
 from __future__ import absolute_import
 from collections import Counter
 from itertools import chain
+from logging import getLogger
 
 from celery.app import app_or_default
 
 from ..utils import KeyDefaultDict
 from . import Proc
 
 
+logger = getLogger(__name__)
+
+
 class CeleryInspector(KeyDefaultDict):
     """
     A defaultdict that manages the celery inspector cache.
     """
 
-    def __init__(self, app):
+    def __init__(self, app, simple_queues=False):
         super(CeleryInspector, self).__init__(self.get_status_task_counts)
         self.app = app
         self.route_queues = None
+        self.simple_queues = False
```

Comment:
You are setting it always to `False` instead of the method argument `simple_queues` ;)